src/pages/setStandards/index.css

```diff
@@ -0,0 +1,35 @@
+.operatingRange {
+  word-spacing: 1rem;
+}
+
+.thresholdValue {
+  word-spacing: 1rem;
+}
+
+.ant-table-thead > tr > th:nth-child(6) {
```

Comment:
why  antd styles are overwritten here

Reply:
Needed to customize the table. 
Now I have added classname, so that it doesn't overwrite the default style on other tables.